### PR TITLE
Update project lists

### PR DESCRIPTION
**Pull request by brian-c**
Updates featured projects on production.

Shows appropriate (for now) projects on the projects listing page.

### CODE REVIEW

**Comment by brian-c** [comment]
Closes #435.

File: app/pages/home.cjsx [hunk 1]
```diff
@@ -10,6 +9,11 @@ ProjectCard = require '../partials/project-card'
 alert = require '../lib/alert'
 LoginDialog = require '../partials/login-dialog'
 
+FEATURED_PRODUCT_IDS = if process.env.NODE_ENV is 'production'
```
Comment (chrissnyder) [inline app/pages/home.cjsx]: Why not expose the env from `app/api/config.coffee` and use whatever comes out of that?

Comment (brian-c) [inline app/pages/home.cjsx]: That'd be nice to develop with, but this way envify will turn `process.env.NODE_ENV` into `"production"`, then uglify will turn `"production" === "production"` into `true` and drop the unreachable code in the build. Not a huge optimization in this case, but probably a good habit to set.

Comment (chrissnyder) [inline app/pages/home.cjsx]: Sounds good to me.

**Pull request merged**